tests/README.md

```diff
@@ -0,0 +1,90 @@
+# Running full e2e test suite in Minikube
+
+This doc describes how to to run the entire suite of e2e tests locally, with no external (cloud) dependencies. It will use minikube, spark k8s operator and minio for storage.
+
+The tests will be run against your local copy of the Python SDK and ingestion jar. For other components like core and serving this setup will use docker images from the public GCR repo, built from the latest master.
+
+## Prerequisites:
+* Docker (highly recommend increasing default disk image size on OSX in docker settings)
+* awscli
+* kubectl
+* minikube (tested using docker driver on OSX)
+* helm 3
+* bash 5.0+ (you'll need to brew install it on OSX)
+* Java 11 toolchain with maven
+* make
+
+## Steps
+
+1. Start minikube. We'll need more memory and disk than default.
+```bash
+minikube start --disk-size='40000mb' --memory 4096
+```
+
+2. Install minio. 
+```bash
+helm repo add minio https://helm.min.io/
+helm install --namespace minio --create-namespace minio minio/minio --set resources.requests.memory=2Gi
+```
+
+3. Create k8s namespace to run tests in
+```bash
+kubectl create namespace sparkop
+```
+
+4. Install spark operator.
+```bash
+helm repo add spark-operator https://googlecloudplatform.github.io/spark-on-k8s-operator
+helm install spark-op spark-operator/spark-operator \
+	--namespace spark-operator \
+	--create-namespace \
+	--set "image.tag=v1beta2-1.1.2-2.4.5" \
+	--set "sparkJobNamespace=sparkop" \
+	--set "serviceAccounts.spark.name=spark"
+```
+
+5. Copy secret from minio into new sparkop namespace.
+```bash
+kubectl get secret minio --namespace=minio -oyaml | grep -v '^\s*namespace:\s' | kubectl apply --namespace=sparkop -f -
+```
+
+6. Install Feast. That may fail due to timeout, rerun again in that case
+```bash
+#
+# NB!! make sure to use bash 5.0+ for the next step. You'd need to brew install it on OSX
+DOCKER_REPOSITORY=gcr.io/kf-feast GIT_TAG=develop bash ./infra/scripts/setuplocal.sh
```

Comment:
Should this be setup-e2e-local.sh?

Reply:
yep good catch, that was a last minute rename